addons/test_mail/static/tests/activity_tests.js

```diff
@@ -658,6 +659,50 @@ QUnit.module("test_mail", {}, function () {
         }
     );
 
+
+    QUnit.test("activity view: activity_ids condition not applied as domain search create dialog", async function (assert) {
+        patchWithCleanup(ActivityController.prototype, {
+            scheduleActivity() {
+                this._super();
+                assert.step(JSON.stringify(this.props.domain));
+            },
+        });
+        Object.assign(serverData.views, {
+            "mail.test.activity,false,list":
+                '<tree string="MailTestActivity"><field name="name"/></tree>',
+        });
+
+        const { env, openView } = await start({
+            serverData,
+        });
+        await openView({
+            res_model: "mail.test.activity",
+            views: [[false, "activity"]],
+        });
+        patchWithCleanup(env.services.action, {
+            doAction(action, options) {
+                assert.step("doAction");
+                options.onClose();
+            },
+        });
+
+        // open search create dialog
```

Comment:
if I revert those 2 lines, the test is green 
```js
onClose: () => this.model.load(getSearchParams(this.props)),
onReloadData: () => this.model.load(getSearchParams(this.props)),
```

do you think we can include them in the test ? (or maybe it's to annoying to do ?)

Reply:
Yes, because the issue related to those 2 lines is created in a different way.
I've added test cases for both the cases.
Thanks : )